x-pack/plugins/case/server/routes/api/cases/configure/get_connectors.ts

```diff
@@ -17,39 +17,16 @@ import {
   RESILIENT_ACTION_TYPE_ID,
 } from '../../../../../common/constants';
 
-/**
- * We need to take into account connectors that have been created within cases and
- * they do not have the isCaseOwned field. Checking for the existence of
- * the mapping attribute ensures that the connector is indeed a case connector.
- * Cases connector should always have a mapping.
- */
-
-interface CaseAction extends FindActionResult {
-  config?: {
-    isCaseOwned?: boolean;
-    incidentConfiguration?: Record<string, unknown>;
-  };
-}
-
-const isCaseOwned = (action: CaseAction): boolean => {
-  if (
-    [SERVICENOW_ACTION_TYPE_ID, JIRA_ACTION_TYPE_ID, RESILIENT_ACTION_TYPE_ID].includes(
-      action.actionTypeId
-    )
-  ) {
-    if (action.config?.isCaseOwned === true || action.config?.incidentConfiguration?.mapping) {
-      return true;
-    }
-  }
-
-  return false;
-};
+const isCaseOwned = (action: FindActionResult): boolean =>
```

Comment:
nit -> maybe we can rename to `isConnectorSupported` now that we removed `action` logic.